src/Symfony/Component/Console/Tester/TesterTrait.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Tester;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Amrouche Hamza <hamza.simperfit@gmail.com>
+ *
+ * @internal
+ */
+trait TesterTrait
+{
+    /**
+     * Gets the display returned by the last execution of the application.
+     *
+     * @param bool $normalize Whether to normalize end of lines to \n or not
+     *
+     * @return string The display
+     */
+    public function getDisplay($normalize = false)
+    {
+        rewind($this->output->getStream());
+
+        $display = stream_get_contents($this->output->getStream());
+
+        if ($normalize) {
+            $display = str_replace(PHP_EOL, "\n", $display);
+        }
+
+        return $display;
+    }
+
+    /**
+     * Gets the input instance used by the last execution of the application.
+     *
+     * @return InputInterface The current input instance
+     */
+    public function getInput()
+    {
+        return $this->input;
+    }
+
+    /**
+     * Gets the output instance used by the last execution of the application.
+     *
+     * @return OutputInterface The current output instance
+     */
+    public function getOutput()
+    {
+        return $this->output;
+    }
+
+    /**
+     * Gets the status code returned by the last execution of the application.
+     *
+     * @return int The status code
+     */
+    public function getStatusCode()
+    {
+        return $this->statusCode;
+    }
+
+    /**
+     * Sets the user inputs.
+     *
+     * @param $inputs array an array of strings representing each input
+     *              passed to the command input stream
+     *
+     * @return self
+     */
+    public function setInputs(array $inputs)
```

Comment:
For this to work in ApplicationTester, the tester needs to set the input stream from this value.
So `CommandTester::createInputStream()` should be shared between both testers (that's my motivation for a trait) and [this line](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Tester/CommandTester.php#L65..L67) should be added to `ApplicationTester::run()`.